crates/bevy_render/src/mesh/mesh.rs

```diff
@@ -204,16 +208,26 @@ impl From<&Indices> for IndexFormat {
 }
 
 // TODO: allow values to be unloaded after been submitting to the GPU to conserve memory
-#[derive(Debug, TypeUuid, Clone)]
+#[derive(Debug, TypeUuid, Clone, Reflect)]
```

Comment:
If Reflect is only being added for your animation system, can we just add it in the animation pr instead? Otherwise this impl seems a bit pointless / I probably wouldn't include it. I think requiring "empty reflect impls" is something thats worth discussing in the animation pr.